privacy.html

```diff
@@ -117,15 +117,6 @@
 
 <style>
```

Comment:
Might make sense to leave some comments about what is happening here so it doesn't get removed in future iteration.